lib/spack/spack/test/cmd/install.py

```diff
@@ -248,11 +248,11 @@ def test_install_overwrite_not_installed(
 
 def test_install_commit(
         mock_git_version_info, install_mockery, mock_packages, monkeypatch):
-    """
-    Test installing a git package from a commit.
+    """Test installing a git package from a commit.
```

Comment:
accidental?

Reply:
Intentional -- see https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

Reply:
Few lines down too?

Reply:
The print? Oops.  It won't be printed in tests though so I'm inclined to leave it for now (or we get to wait for the vis SDK again -- hoping that CI will improve soon)